src/base/bittorrent/trackerentry.cpp

```diff
@@ -72,6 +76,33 @@ void TrackerEntry::setTier(const int value)
     m_nativeEntry.tier = value;
 }
 
+int TrackerEntry::numSeeds() const
+{
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    return nativeEntry().scrape_complete;
+#else
+    return nativeEntry().endpoints.empty() ? -1 : nativeEntry().endpoints[0].scrape_complete;
```

Comment:
there can be more than 1 endpoints in libtorrent 1.2??
should we get the max number of all endpoints instead of only the first one?

Reply:
>there can be more than 1 endpoints in libtorrent 1.2??

AFAIK, libtorrent-1.1 allows multiple listen interfaces too. libtorrent-1.2 just differentiate tracker requests/responses by listen socket.
In fact, this is one of the problematic issues, and I don't know yet what to do with it, how we should handle them all. So I've used a simple workaround here for now. We will need to fix it before libtorrent-1.2 support enters the release stage.

Reply:
OK, perhaps you should add an comment or `#pragma message` so we don't forgot it?

Reply:
Done.